librz/core/core.c

```diff
@@ -1718,34 +1720,6 @@ RZ_API void rz_core_autocomplete(RZ_NULLABLE RzCore *core, RzLineCompletion *com
 			}
 		}
 		rz_list_free(vars);
-	} else if (!strncmp(buf->data, "t ", 2) || !strncmp(buf->data, "t- ", 3)) {
```

Comment:
I removed that type and `pf` autocompletion code since we are switching to the newshell anyway.